### ISSUE TITLE
line break not working on WSL2

### ISSUE THREAD

**Issue by mattocci27** [randy3k/radian#225]
- radian version: 0.5.8
- r version: 4.0.2
- Ubuntu 20.4 on WSL2

Copying and pasting this 

```
rnorm(3,
  1,
  0)
```

to an radian console on WSL2 ubuntu results in the following output.

```
r$> rnorm(3,)
[1] -0.225570560 -0.003819365  0.330800087

r$>   1,
Error: unexpected ',' in "  1,"

r$>   0)
Error: unexpected ')' in "  0)"
```

The above command works on an R console on WSL2 ubuntu and an radian console on a local ubuntu (not in WSL). Any suggestions??

**Comment by randy3k** [comment]
What app are you using? It doesn't seem that it supports bracketed paste mode.

**Comment by mattocci27** [comment]
Yes, that's true. It seems the problem of a terminal side. Line break doesn't work on Windows terminal but it works on Alacritty, and terminal in VS Code. Thanks!

**Comment by mr-majkel** [comment]
I can confirm this is an issue with radian on Windows 10, with Neovim-Qt and Goneovim built-in terminals. R.exe doesn't have this issue.

**Comment by randy3k** [comment]
By the way, you could turn off auto match to make things behave better.
